paper-checkbox.html

```diff
@@ -164,12 +170,17 @@
         position: relative;
         display: inline-block;
         vertical-align: middle;
-        padding-left: 8px;
+        padding-left: var(--paper-checkbox-label-spacing, 8px);
         white-space: normal;
         pointer-events: none;
         color: var(--paper-checkbox-label-color, --primary-text-color);
       }
 
+      :host-context([dir="rtl"]) #checkboxLabel {
+        padding-right: var(--paper-checkbox-label-spacing, 8px);
+        padding-left: 0;
```

Comment:
I guess "whatever you are normally doing" doesn't have a mixin in this case (should it?), so no mixin is required for `rtl`.